teuthology/run.py

```diff
@@ -249,9 +249,11 @@ def get_initial_tasks(lock, config, machine_type):
             {'internal.git_ignore_ssl': None},
             {'internal.setup_cdn_repo': None},
             {'internal.setup_base_repo': None},
-            {'internal.setup_additional_repo': None},
-            {'kernel.install_latest_rh_kernel': None}
+            {'internal.setup_additional_repo': None}
         ])
+        # Install latest kernel task for redhat downstream runs
+        if config.get('redhat').get('install_latest_rh_kernel', False):
```

Comment:
In the email i saw a comment from @kshtsk  but not here, I think we need to handle when 'redhat' is none.

Reply:
well the diff is confusing, the outer block has check so its safe.